packages/frontend/components/save-post/image-input.tsx

```diff
@@ -78,6 +78,11 @@ function ImageInput({ images, setImages }: ImageInputProps) {
     setCurrentImage(undefined);
   };
 
+  const onImageDelete = (img: string) => {
+    setImagePreviews(imagePreviews.filter((item) => item !== img));
```

Comment:
do we handle image deleting from server if it was already uploaded to storage?

Reply:
yep